scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -198,9 +200,16 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     case Tokenized.Error(_, _, details) => throw details
   }
   lazy val (parserTokens, parserTokenPositions) = {
+    def indentedRegion(regions: List[SepRegion]): Boolean =
+      regions.headOption.exists { case SepRegion(c, _) => c == 'O' || c == 'E' }
```

Comment:
Let's maybe add a legend what each sign means.

Reply:
Changed this char to Enum which should be much more descriptive :+1: